advanced_communication_options/update_comms_settings.py

```diff
@@ -125,8 +196,8 @@ def check_pi3():
 	if hw_id=="BCM2709":
```

Comment:
Looks like it.  The Pi I have does have the bluetooth/wifi antenna on it and it's detected as a Pi2